cirq/ops/common_gates.py

```diff
@@ -665,6 +679,12 @@ def _qasm_(self, args: protocols.QasmArgs,
     def _value_equality_values_(self):
         return self.num_qubits(),
 
+    def _json_dict_(self):
+        return {
+            'cirq_type': self.__class__.__name__,
+            'num_qubits': self._num_qubits
```

Comment:
should num_qubits be a property or an attribute?

Reply:
`num_qubits()` is a function. Most other gates do not take `num_qubits` as an argument to the constructor because most other gates are for a fixed number of qubits

Reply:
Changing `self._num_qubits` -> `self.num_qubits()` to be more obvious